clang/test/CodeGenSYCL/kernel-arg-accessor-pointer.cpp

```diff
@@ -0,0 +1,122 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
+// This test checks if the metadata "kernel-arg-runtime-aligned"
+// is generated if the kernel captures an accessor
+
+#include "sycl.hpp"
+
+using namespace cl::sycl;
+
+queue q;
+
+int main() {
+
+  using Accessor =
+      accessor<int, 1, access::mode::read_write, access::target::global_buffer>;
+  Accessor acc[2];
+
+  accessor<int, 1, access::mode::read, access::target::global_buffer> readOnlyAccessor;
+
+  cl::sycl::accessor<float, 2, cl::sycl::access::mode::write,
```

Comment:
self review: will get rid of `cl::sycl` on all applicable instances,  since we have `using namespace cl::sycl;`